packages/pg/collection.js

```diff
@@ -56,6 +56,32 @@ const QBProto = Meteor.isClient ?
   Knex.QueryBuilder.prototype :
   Knex.Client.prototype.QueryBuilder.prototype;
 
+// Addresses https://github.com/meteor/postgres-packages/issues/9
+QBProto.fetchOne = function fetchOne() {
+  return QBProto.fetch.call(this)[0];
```

Comment:
Should we put in an effort to throw a nicer error when no records were returned?